### ISSUE TITLE
Add currentCommit to upload endpoint

### ISSUE THREAD

**Issue by laurentsigal** [waldoapp/waldo-cli#11]
With the short shas we have the information we need in order to resolve history between different app versions.
The goal of the `currentCommit` is to represent the last remote commit. It is not necessarily the latest commit of the current log because of cases like this:
```
git pull origin/feature-1
git rebase master
```
even worse for us could be something like:
```
git pull origin/feature-1
echo `CI_RUN=true`
git commit --amend
```
So the best way to get this first remote commit hash is either:
- get from CI environment the hash. this might very well be exposed in most ci tools because that's also something that they use
- get from log the last line pointing to remote, hoping we're not in the second case.

**Comment by eBardX** [comment]
That first method (get the hash from the CI environment) seems pretty iffy.

For example, Bitrise defines `BITRISE_GIT_COMMIT` as

> The commit hash of the Git commit that triggered the build, when applicable. There is no commit hash, for example, in the case of a pull request build.

In addition, a quick test where the build is manually triggered, results in that environment variable not being defined either.

**Comment by eBardX** [comment]
Implemented in `Waldo CLI 1.4.2`.
